packages/web-components/src/components/footer/locale-button.ts

```diff
@@ -25,6 +25,12 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-locale-button`)
 class DDSLocaleButton extends StableSelectorMixin(FocusMixin(LitElement)) {
+  /**
+   * Size property to apply different styles.
+   */
+  @property()
+  size?: string;
```

Comment:
Could you make this an enum? Ref: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/src/components/footer/footer.ts#L48